terrain_navigation_ros/config/ap_config.yaml

```diff
@@ -0,0 +1,273 @@
+# Common configuration for ArduPilot
+#
+# node:
+/mavros/**:
+  ros__parameters:
+    startup_px4_usb_quirk: false
+
+# --- system plugins ---
+
+# sys_status & sys_time connection options
+/mavros/**/conn:
+  ros__parameters:
+    heartbeat_rate: 1.0    # send heartbeat rate in Hertz
+    heartbeat_mav_type: "ONBOARD_CONTROLLER"
+    timeout: 10.0          # heartbeat timeout in seconds
+    timesync_rate: 10.0    # TIMESYNC rate in Hertz (feature disabled if 0.0)
+    system_time_rate: 1.0  # send system time to FCU rate in Hertz (disabled if 0.0)
+
+# sys_status
+/mavros/**/sys:
+  ros__parameters:
+    min_voltage: [10.0]   # diagnostics min voltage, use a vector i.e. [16.2, 16.0] for multiple batteries, up-to 10 are supported
+                        # to achieve the same on a ROS launch file do: <rosparam param="sys/min_voltage">[16.2, 16.0]</rosparam>
+    disable_diag: false # disable all sys_status diagnostics, except heartbeat
+
+# sys_time
+/mavros/**/time:
+  ros__parameters:
+    time_ref_source: "fcu"  # time_reference source
+    timesync_mode: MAVLINK
+    timesync_avg_alpha: 0.6 # timesync averaging factor
+
+# --- mavros plugins (alphabetical order) ---
+
+# 3dr_radio
+/mavros/**/tdr_radio:
+  ros__parameters:
+    low_rssi: 40  # raw rssi lower level for diagnostics
+
+# actuator_control
+# None
+
+# command
+/mavros/**/cmd:
+  ros__parameters:
+    use_comp_id_system_control: false # quirk for some old FCUs
+
+# dummy
+# None
+
+# ftp
+# None
+
+# global_position
+/mavros/**/global_position:
+  ros__parameters:
+    frame_id: "map"             # origin frame
```

Comment:
Comments are good, but do you want to remove all the extra spaces between code and comments?